tests/Aspire.EndToEnd.Tests/IntegrationServicesTests.cs

```diff
@@ -35,6 +35,7 @@ public IntegrationServicesTests(ITestOutputHelper testOutput, IntegrationService
     [InlineData(TestResourceNames.sqlserver)]
     [InlineData(TestResourceNames.efsqlserver)]
     [InlineData(TestResourceNames.milvus)]
+    [InlineData(TestResourceNames.elasticsearch)]
```

Comment:
Now that we have "FunctionalTests" in ElasticsearchFunctionalTests, I don't think we need to add this to the EndToEnd tests.

Using the new separate Hosting tests pattern, along with functional tests there, means we don't need to have every resource/component pair in the EndToEnd tests. Instead we can just use some "core" resources and components in the EndToEnd, and the other resources can be tested in their own assembly.

cc @radical @mitchdenny @sebastienros